### ISSUE TITLE
Add 'contributor' in addition to original author(s)

### ISSUE THREAD

**Issue by droumis** [holoviz-topics/examples#456]
Let's consider adding a 'contributor' field in addition to the original author(s) to encourage smaller maintenance/updates and give credit to those who made contributions to some targeted aspects but not quite at the level of an author. This approach has been adopted by [Pythia](https://projectpythia.org/cmip6-cookbook/README.html), for instance.

**Comment by droumis** [comment]
Alternatively, [Galaxy](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/tutorial.html) calls such contributors 'Editors'

![Image](https://github.com/user-attachments/assets/bc1e74de-3d28-46f6-9ac6-8e2b20ce1d6a)

**Comment by jbednar** [comment]
Good point! For the specific case of an example, I'd reflexively read "contributor" as being synonymous with "original author" -- the person  who contributed the example. That's not what's meant here, just someone who contributes _to_ the example. It would be more clear if it says "Other contributor(s)" to signify being in contrast to "authors", though that's wordy. I think "Editor" is also a bit odd; sounds like either a copyeditor or a scientific article editor, neither of which likely applies here. And "Maintainer" to me implies an ongoing role that also doesn't necessarily apply. So I don't love any of those three words to indicate "someone who changed these files in some way not amounting to authorship", but I don't have any better suggestion and I do like the idea of being able to distinguish, so I'm happy with whatever!
